src/ipa_pc/data_structures.rs

```diff
@@ -127,7 +126,7 @@ impl<G: AffineCurve> PCCommitment for Commitment<G> {
     }
 
     fn size_in_bytes(&self) -> usize {
-        to_bytes![G::zero()].unwrap().len() / 2
+        (to_bytes![G::zero()].unwrap().len() / 2) * self.comm.len()
```

Comment:
I expect that both, the previous as well as the updated formulas, are just approximate and refer to the size of a commitment  using compressed representation of elliptic curve points. Why not using the ToCompressedBits trait for AffineCurve?

Reply:
I think that's because arkworks write implementation for G already does the compression. For ginger-lib this is not true, but soon enough we're going to implement it in this way, so we can leave this function as it is for the moment (it's used just to print info btw)